br/pkg/restore/batcher_test.go

```diff
@@ -49,7 +49,7 @@ func (sender *drySender) Close() {
 }
 
 func waitForSend() {
-	time.Sleep(10 * time.Millisecond)
+	time.Sleep(50 * time.Millisecond)
```

Comment:
Do we need to set 50ms in the production and set 10ms in the test？ I am afraid that Too long timeout will affect CI.

Reply:
@3pointer @lance6716 PTAL

Reply:
@hawkingrei The `waitForSend` will be called 4 times in total, compared with the cost of unit-test failure, I think the cost of increasing the sleep time is relatively small, but maybe we can double the sleep time, that is, 20ms.

Reply:
The `waitForSend` is only used for test.